changelogs/unreleased/config-anonymous-replica.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/config
+
+* Support `replication.anon` option (gh-9432).
```

Comment:
```suggestion
* Introduced the `replication.anon` configuration option (gh-9432).
```

Reply:
In fact, the option was there, but an attempt to use it would reveal existing problems.

Reply:
Moreover, 'introduced' is suggested for significant features in our changelog writing guide. I wouldn't qualify it so.

Reply:
We discussed it f2f and agreed on the following wording.

> * Effectively supported `replication.anon` option, which was broken in several ways before (gh-xxxx).

Reply:
Updated. Thanks!